### PR TITLE
feat: `viur install update` command

### PR DESCRIPTION
**Pull request by Grashalmbeisser**
Updates all ViUR components with one command

### CODE REVIEW

File: src/viur_cli/install.py [hunk 1]
```diff
@@ -81,6 +82,36 @@ def install():
...
+            echo_info(f"updating {build}...")
+            output = subprocess.run(f"viur install {build} latest {profile}",
+                                    capture_output=True, check=True, shell=True)
+            checkreturncode(output)
+
+    # This function is obsolete, howerver the user might expect this behaviour.
+    else:
+        output = subprocess.run(f"viur install {action} latest {profile}",
+                                capture_output=True, check=True, shell=True)
+        checkreturncode(output)
```
Comment (sveneberth) [inline src/viur_cli/install.py:112]:
```suggestion
@install.command()
@click.argument("action", type=click.Choice(["vi", "admin", "scriptor", "all"]))
@click.argument("profile", default='default')
def update(action, profile):
    """Updates all ViUR Ecosystem Components"""
    conf = config.get_profile(profile)
   
    def run_single_action(action: str) -> None:
        echo_info(f"Updating {build} …")
        output = subprocess.run(f"viur install {action} latest {profile}",
                                capture_output=True, check=True, shell=True)
        if output.returncode == 0:
            echo_positive("update was successful")
        else:
            echo_fatal("update exited with a non zero success code")
    
    if action == "all":
        # Get the builds of the project.json profile
        for build in conf["builds"]:
            run_single_action(build)
    # This function is obsolete, however the user might expect this behavior.
    else:
        run_single_action(action)
```

**Review by sveneberth — CHANGES_REQUESTED** [review]

**Comment by sveneberth** [comment]
The command name is IMO not the best. `viur install update` means you install the component `update`, not you perform an update.

**Comment by Grashalmbeisser** [comment]
> The command name is IMO not the best. `viur install update` means you install the component `update`, not you perform an update.

You have a point, i refactored everything again and the command group is now "package"

File: README.md [hunk 2]
```diff
@@ -67,9 +67,16 @@ create a backup bucket and enable the gcloud service worker account to access it
...
 ```
-ask viur-cli to install either vi or the scriptor into your project
+handles viur ecosystem package operations
+
+Commands:
+- `install`  Installes a viur package
+- `update`   updates a viur package
+Arguments:
+- `profile`  profile you want to install to 
+- `version`  version you want to install
```
Comment (phorward) [inline README.md:79]:
```suggestion
handles ViUR ecosystem package operations

Commands:
- `install`  installs a ViUR package (in a specific version)
- `update`   updates a ViUR package to the newest version

Arguments:
- `profile`  profile to install to
- `version`  version to install
```

**Review by phorward — CHANGES_REQUESTED** [review]
Okay for the first version. I think there's still stuff which can be optimized.

**Pull request merged**